addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua

```diff
@@ -1,344 +1,412 @@
-	language.Add( "tool.wire_joystick.0", "Primary: Create/Update Joystick   Secondary: Copy Settings    Reload: Link to pod" )
-	language.Add( "tool.wire_joystick.1", "Now select the pod to link to, or anything other than a pod to revert.")
-	language.Add( "WirejoystickTool_joystick", "Joystick:" )
-	language.Add( "sboxlimit_wire_joysticks", "You've hit the Joysticks limit!" )
-	language.Add( "undone_Wire Joystick", "Undone Wire Joystick" )
+
+  TOOL.Information = {
+    { name = "info"   , stage = 1},
+    { name = "left"  },
+    { name = "right" },
+    { name = "reload"}
+  }
+
+  language.Add( "tool."..gsToolModeOP..".name"           , "Joystick Tool (Wire)" )
+  language.Add( "tool."..gsToolModeOP..".desc"           , "Spawns a Joystick Module interface chip for use with the wire system" )
+  language.Add( "tool."..gsToolModeOP..".left"           , "Create / Update joystick" )
+  language.Add( "tool."..gsToolModeOP..".right"          , "Copy joystick settings. Hit world to open configuration" )
+  language.Add( "tool."..gsToolModeOP..".reload"         , "Link joystick to pod controller" )
+  language.Add( "tool."..gsToolModeOP..".1"              , "Now select the pod to link to, or anything other than a pod to revert.")
+  language.Add( "tool."..gsToolModeOP..".uid"            , "Unique identifier. No spaces, alphanumeric, 17 character limit!" )
+  language.Add( "tool."..gsToolModeOP..".uid_con"        , "UID")
+  language.Add( "tool."..gsToolModeOP..".autofill"       , "Write a positive number here and hit ENTER to trigger random text autofill")
+  language.Add( "tool."..gsToolModeOP..".description"    , "Write some input description here. Maximum 20 characters! For example `Steering`" )
+  language.Add( "tool."..gsToolModeOP..".description_con", "Description")
+  language.Add( "tool."..gsToolModeOP..".lcontr"         , "This labels the given set of input control configuration settings" )
+  language.Add( "tool."..gsToolModeOP..".lcontr_con"     , "Control configuration:" )
+  language.Add( "tool."..gsToolModeOP..".maxon"          , "Maximum output value when analogue or ON value when digital" )
+  language.Add( "tool."..gsToolModeOP..".maxon_con"      , "Maximum / On")
+  language.Add( "tool."..gsToolModeOP..".minoff"         , "Minimum output value when analogue or OFF value when digital" )
+  language.Add( "tool."..gsToolModeOP..".minoff_con"     , "Minimum / Off" )
+  language.Add( "tool."..gsToolModeOP..".analog"         , "Enable this when your source is analogue input" )
+  language.Add( "tool."..gsToolModeOP..".analog_con"     , "Analog input" )
+  language.Add( "tool."..gsToolModeOP..".config"         , "Click this button to open the joystick configuration. You can also right click on the world" )
+  language.Add( "tool."..gsToolModeOP..".config_con"     , "Joystick Configuration" )
+  language.Add( "undone_"..gsToolModeOP                  , "Undone Wire Joystick !" )
```

Comment:
Why is there a space between the word and exclamation point?

Reply:
Revered in 97acdb1cc9315998cdd73092fceeb1dfbf1f8afc